### DISCUSSION TITLE
Will data from the James Webb Space Telescope disprove the Big Bang? Find out on Manifold

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#169] [General]
Just for fun, it's interesting to read what people think on Manifold [about cosmology](https://manifold.markets/BTE/will-data-from-the-james-webb-space)
This market cites Eric Lerner's 2022-8 article on IAI, but there is not much discussion.

There's also "[Will data from the James Webb Space Telescope increase the Hubble Tension?](https://manifold.markets/Shump/will-data-from-the-james-webb-space-f8ec530fbd83)".

Both questions are kind of trivial.  They will be resolved by mainstream astronomers... but can a few dissidents 'influence the market' and make 'mana'? 😉

**Comment by OlliSant** [comment]
That is a Good Question! I think the galaxies far away in time and space clearly and definitely disproves the BB. Why they do not see this is the strange thing.

**Reply by RedshiftDrift** [reply]
Many people believe that you can obtain knowledge from a large number of ignorant people.  This absurd idea is what drives "prediction markets" like [Manifold](https://github.com/orgs/a-cosmology-group/discussions/169#discussion-5715972).

As for why cosmologists can't see that distant galaxies are incompatible with the Big Bang model, it's also a psychological factor.  They have [so much invested in it](https://en.wikipedia.org/wiki/Sunk_cost#Fallacy_effect) they blame the problems on something else (*"[our previously-favored picture of galaxy formation in the early universe must be revised](https://www.axios.com/2023/01/17/james-webb-space-telescope-early-galaxies)"*).

**Reply by OlliSant** [reply]
And taking more "epicycles" it is always possible to go on with the old theory. But not eternally in the science. It is time to change the theory. Who makes the better one? ACG?

**Comment by bgreenman** [comment]
BB theory represents the scientific mainstream world view and its refutation will go far beyond mainstream astronomy and physics

**Comment by OlliSant** [comment]
Can you give me a link, where the "baby galaxies" in Hubble are seen as big spiral galaxies in Webb telescope view?

**Reply by OlliSant** [reply]
I know there is a link but I don't find it

**Reply by OlliSant** [reply]
https://petapixel.com/2022/07/11/comparing-hubble-to-james-webb-the-difference-in-detail-is-astounding/
This is one in the beginning already.
